scripts/demo-fruits.sh

```diff
@@ -10,51 +10,18 @@ source ${SCRIPTS_DIR}/play-demo.sh
 ####################################
 VM_IP=${VM_IP:=127.0.0.1}
 NAMESPACE=app
-REGISTRY=localhost:5000
-IMAGE_VERSION=latest
+IMAGE=quay.io/halkyonio/atomic-fruits:latest
 INGRESS_HOST=atomic-fruits.${VM_IP}.nip.io
-QUARKUS_APP_PATH="$HOME/atomic-fruits-service"
-GITHUB_REPO=https://github.com/aureamunoz/atomic-fruits-service
-
-# Database credential
-DB_NAMESPACE=db
-USERNAME=healthy
-PASSWORD=healthy
-TYPE=postgresql
 
 # Parameters to play the demo
 TYPE_SPEED=${TYPE_SPEED:=200}
 NO_WAIT=true
 
-if [[ ! -d "${QUARKUS_APP_PATH}" ]]; then
-  git clone ${GITHUB_REPO} ${QUARKUS_APP_PATH}
-fi
-
-if [[ "$1" == "clean" ]]; then
-  pe "helm uninstall postgresql -n ${DB_NAMESPACE} || true"
-  pe "k delete -f ${QUARKUS_APP_PATH}/target/kubernetes/kubernetes.yml -n ${NAMESPACE} || true"
-  pe "k delete ns ${NAMESPACE} || true"
-  pe "k delete ns ${DB_NAMESPACE} || true"
-  exit 0
-fi
-
-pushd ${QUARKUS_APP_PATH}
-
 pe "k create ns $NAMESPACE --dry-run=client -o yaml | k apply -f -"
 pe "k config set-context --current --namespace=${NAMESPACE}"
 
-pe "helm repo add bitnami https://charts.bitnami.com/bitnami"
-pe "helm install postgresql bitnami/postgresql \
```

Comment:
How will the DB been installed and parameters defined ?

Reply:
The database is installed as a Helm chart dependency when installing the `fruits-app` chart.
And the parameters can be defined using `--set db.auth.database=xx --set db.auth.username=xx --set db.auth.password=xx`. If not provided, it will use the defaults.

Reply:
`The database is installed as a Helm chart dependency when installing the fruits-app chart.` Is it documented as currently nobody knows that such helm dependency is included part of the helm generated from this project : https://github.com/halkyonio/atomic-fruits-service except if you look to the code ?

Here to define the dependency and to configure the DB: user, pwd and database name : https://github.com/halkyonio/atomic-fruits-service/blob/main/src/main/resources/application.properties#L27-L32

Here to declare the db parameters: https://github.com/halkyonio/atomic-fruits-service/blob/main/src/main/resources/application.properties#L33-L36
?

Reply:
Your links are pointing to the `application.properties` file, which is not directly exposed to the Helm chart. 
As we're using the Helm chart, this is documented in https://github.com/halkyonio/helm-charts#fruits-app.

Reply:
As soon as we will have a proper README and table of the parameters, that will improve to discover what it is possible to customize with the chart genrated